oaknational/Oak-Web-Application
# feat: add ability to mark up blog posts with footnotes

## Description

Enables marking up blog posts with footnotes in the CMS

Some A11Y feedback would be appreciated, specifically how the footnotes themselves should read (currently the extra labelling attributes are just on the <sup>N</sup>s

## How to test

1. Go to the [testing blog post in preview mode](https://deploy-preview-1670--oak-web-application.netlify.thenational.academy/api/preview/blog/some-blog-post?secret=wsJ3DpANkjZQk9z)
2. Search for "footnote", check both work and the backlinks work
3. [Other blog posts](https://deploy-preview-1670--oak-web-application.netlify.thenational.academy/blog/our-strategy-update) should not contain the "references" section

n.b. footnotes can be modified [in the CMS here](https://cms.thenational.academy/desk/webinarsBlogs;blogPosts;2ed3149d-8c7d-41cb-9123-12ff405ff979)

## Screenshots
Inline:
![CleanShot 2023-06-01 at 12 55 19](https://github.com/oaknational/Oak-Web-Application/assets/2717635/8c7884a4-8c8b-4212-823c-fb8597236dca)

The end of the blog post:
![CleanShot 2023-06-01 at 12 55 11](https://github.com/oaknational/Oak-Web-Application/assets/2717635/12d71838-36f8-43f9-a2de-5f13afff48d9)

## Checklist

    Attempted to mark up the footnotes as best I could for A11Y, but open to feedback
